packages/nextjs/pages/api/og.tsx

```diff
@@ -0,0 +1,205 @@
+import { NextRequest } from "next/server";
+import { ImageResponse } from "@vercel/og";
+import { Address, createPublicClient, http } from "viem";
+import { mainnet } from "viem/chains";
+
+export const publicClient = createPublicClient({
+  chain: mainnet,
+  transport: http(),
+});
+
+export const contractAbi = [
+  {
+    constant: true,
+    inputs: [],
+    name: "name",
+    outputs: [{ name: "", type: "string" }],
+    payable: false,
+    stateMutability: "view",
+    type: "function",
+  },
+] as const;
+
+type NetworkNames = {
+  [key: number]: string;
+};
+
+const networkNames: NetworkNames = {
+  1: "Ethereum Mainnet",
+  11155111: "Sepolia",
+  10: "Optimism",
+  137: "Polygon Mainnet",
+  80001: "Polygon Mumbai",
+  42161: "Arbitrum One",
+  100: "Gnosis Chain",
+  // ... include other network names and their chain IDs here
+};
```

Comment:
Maybe instead of hardcoding we could use logic `NetwroksDropDown`  ?

Reply:
Done at [https://github.com/BuidlGuidl/abi.ninja/pull/90/commits/9fed986f9d849d874de9f201e52ff5713840e836](https://github.com/BuidlGuidl/abi.ninja/pull/90/commits/9fed986f9d849d874de9f201e52ff5713840e836)

Not the exact way you wanted, but let me know if you want me to change it, tested and it works!